.gitignore

```diff
@@ -1 +1,3 @@
 /vendor/
+assets
+.gitignore
```

Comment:
```suggestion
.gitignore

```

Reply:
Meh, what's the reason behind this?

Reply:
Don't forget the empty line too, can't add it on suggestion

Reply:
> Meh, what's the reason behind this?
Yes sorry I'm going to fix that right now